roles/rust-foundation-project-director.md

```diff
@@ -26,8 +33,25 @@ Please refer to [Rust Foundation Director Roles & Responsibilities][director-rol
 - A keen sense of the needs of Rust Project, its teams, processes, and ecosystem.
 - Willingness to understand and represent viewpoints they may not have personal experience with or necessarily agree with.
 
+## Requirements and Eligibility
```

Comment:
I believe that there is also a requirement to be in "good standing" with the project. I think this doesn't have a formal definition, but loosely has meant something akin to not being under any kind of moderation sanction. It might be hard to pin down a definition that would be satisfying in legal contexts, but I don't think that's necessary so perhaps we should try to sketch something out?

Reply:
I added "Project directors must be members of the Rust Project in good standing" with some extra detail about moderation sanction.

I know we're leaving a certain amount of ambiguity with that, but I think this still provides more clarity and we can better define "good standing" if needed in the future.